arm/ui/routes.py

```diff
@@ -89,17 +100,29 @@ def submitrip():
 def changeparams():
     config_id = request.args.get('config_id')
     config = Config.query.get(config_id)
+    app.logger.debug(config.pretty_table())
+    job = Job.query.get(config_id)
     form = ChangeParamsForm(obj=config)
     if form.validate_on_submit():
         config.MINLENGTH = format(form.MINLENGTH.data)
         config.MAXLENGTH = format(form.MAXLENGTH.data)
         config.RIPMETHOD = format(form.RIPMETHOD.data)
-        #config.MAINFEATURE = format(form.MAINFEATURE.data)
+        # config.MAINFEATURE = int(format(form.MAINFEATURE.data) == 'true')
+        config.MAINFEATURE = bool(format(form.MAINFEATURE.data))  # must be 1 for True 0 for False
+        app.logger.debug(f"main={config.MAINFEATURE}")
+        job.disctype = format(form.DISCTYPE.data)
         db.session.commit()
-        flash('Parameters changed. Rip Method={}, Main Feature={}, Minimum Length={}, Maximum Length={}'.format(form.RIPMETHOD.data, form.MAINFEATURE.data, form.MINLENGTH.data, form.MAXLENGTH.data))
+        flash(
```

Comment:
I'm not sure if this is nit-picking, but could we move the 'flash' to below the db.session.refresh's, and have the Flash use the actual values in the job and config?

Reply:
Ive commited these changes.